.github/workflows/main.yml

```diff
@@ -11,6 +11,8 @@ on:
   pull_request:
   push:
     branches: [main]
+  schedule:
+    - cron: "0 12 * * SUN"
```

Comment:
Midday on Sunday is quiet for us, but [not quiet for GitHub](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule):

> The `schedule` event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. If the load is sufficiently high enough, some queued jobs may be dropped. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.

As Simon suggested,[^1] could you change the minute field to something less regular?

[^1]: https://bennettoxford.slack.com/archives/C01SE1WB6AW/p1715243059773059?thread_ts=1715242805.627889&cid=C01SE1WB6AW